### PR TITLE
Return statement missing for renderScene

### PR DESCRIPTION
**Pull request by scarlac**
Explain the **motivation** for making this change. What existing problem does the pull request solve?
Example code for "Using Navigators" under "THE BASICS" does not work. renderScene needs to return the components, not just instantiate them.

**Test plan (required)**

I copy-paste and ran the code but did not get anything rendered. I added a return statement before the component which made it work.

Arrow functions need a return statement when supplying a { block of code }.

### CODE REVIEW

**Comment by facebook-github-bot-4** [comment]
By analyzing the blame information on this pull request, we identified @hramos and @gregjacobson to be potential reviewers.

**Comment by facebook-github-bot-4** [comment]
Thank you for your pull request and welcome to our community. We require contributors to sign our Contributor License Agreement, and we don't seem to have you on file. In order for us to review and merge your code, please sign up at https://code.facebook.com/cla - and if you have received this in error or have any questions, please drop us a line at cla@fb.com. Thanks!

**Comment by facebook-github-bot-6** [comment]
Thank you for signing our Contributor License Agreement. We can now accept your code for this (and any) Facebook open source project. Thanks!

File: docs/UsingNavigators.md [hunk 1]
```diff
@@ -105,32 +105,34 @@ A more complete example that demonstrates the pushing and popping of routes coul
 ```javascript
 import React, { Component, PropTypes } from 'react';
 import { Navigator, Text, TouchableHighlight, View } from 'react-native';
-
+r
```
Comment (hramos) [inline docs/UsingNavigators.md]: Looks unintentional

File: docs/UsingNavigators.md [hunk 2]
```diff
@@ -78,7 +78,7 @@ render() {
     <Navigator
       initialRoute={{ title: 'My Initial Scene', index: 0 }}
       renderScene={(route, navigator) => {
-        <MyScene title={route.title} />
+        return <MyScene title={route.title} />
```
Comment (hramos) [inline docs/UsingNavigators.md]: Here we can remove the curly braces instead, since it is a single line component.

File: docs/UsingNavigators.md [hunk 3]
```diff
@@ -105,32 +105,34 @@ A more complete example that demonstrates the pushing and popping of routes coul
...
-            }}
-
-            // Function to call to go back to the previous scene
-            onBack={() => {
-              if (route.index > 0) {
-                navigator.pop();
-              }
-            }}
-          />
+          return (
```
Comment (hramos) [inline docs/UsingNavigators.md]: Is this needed here? We're not using curly braces.

Comment (scarlac) [inline docs/UsingNavigators.md]: I looked at the two other examples in the beginning of the guide to keep the code style the same and they wrap JSX components in parenthesis, even for 1-liners so I tried to keep the style identical. @hramos but I can change it if you want?

File: docs/UsingNavigators.md (hunk 1 above)
Comment (scarlac) [inline docs/UsingNavigators.md]: @hramos Yes, that was unintentional. Sorry about that.

File: docs/UsingNavigators.md (hunk 2 above)
Comment (scarlac) [inline docs/UsingNavigators.md]: Sure, that's a question of preference. I just tried to make as minimal changes as I could. Should I fix it up, @hramos ?

File: docs/UsingNavigators.md (hunk 3 above)
Comment (hramos) [inline docs/UsingNavigators.md]: Fair enough, let's err on being consistent within the same file.

**Comment by facebook-github-bot-9** [comment]
@scarlac updated the pull request.

**Comment by hramos** [comment]
@facebook-github-bot shipit

**Comment by facebook-github-bot-7** [comment]
Thanks for importing.

**Pull request closed without merging**